aeron-driver/src/main/java/io/aeron/driver/MinMulticastFlowControl.java

```diff
@@ -140,7 +154,12 @@ public long onIdle(final long timeNs, final long senderLimit, final long senderP
             receivers = truncateReceivers(receivers, removed);
         }
 
-        return receivers.length > 0 ? minLimitPosition : senderLimit;
+        return receivers.length < groupMinSize || receivers.length == 0 ? senderLimit : minLimitPosition;
+    }
+
+    public boolean hasRequiredReceivers()
```

Comment:
receivers has to be volatile because hasRequiredReceivers is called from the conductor thread whereas the other methods are called from the sender thread.

Reply:
I think groupMinSize also needs to be volatile.